src/chrome/content/rules/downforeveryoneorjustme.com.xml

```diff
@@ -0,0 +1,26 @@
+<!--
+	Non-functional subdomains:
+
+		- www	(m)
+
+	e: expired certificate
+	h: http redirect
+	i: invalid certificate chain
+	m: certificate mismatch
+	r: connection refused
+	s: self-signed certificate
+	t: timeout on https
+-->
+<ruleset name="Down For Everyone Or Just Me">
+
+	<target host="downforeveryoneorjustme.com" />
+	<target host="www.downforeveryoneorjustme.com" />
+	<target host="staging.downforeveryoneorjustme.com" />
+
+	<!-- certificate mismatch -->
```

Comment:
This comment is not needed since you already put this info in the top comment. It should be removed.